source/Calamari.Terraform/PlanTerraformConvention.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using Calamari.Deployment;
+using Calamari.Integration.FileSystem;
+using Calamari.Integration.Substitutions;
+
+namespace Calamari.Terraform
+{
+    public class PlanTerraformConvention : TerraformConvention
+    {
+        private readonly string extraParameter;
+
+        public PlanTerraformConvention(ICalamariFileSystem fileSystem, IFileSubstituter fileSubstituter, string extraParameter = "") : base(fileSystem, fileSubstituter)
+        {
+            this.extraParameter = extraParameter;
+        }
+
+        protected override void Execute(RunningDeployment deployment, Dictionary<string, string> environmentVariables)
+        {
+            string results;
+            using (var cli = new TerraformCLIExecutor(fileSystem, deployment, environmentVariables))
+            {
+                var resultCode = cli.ExecuteCommand(out results, out var commandResult, "plan", "-no-color", "-detailed-exitcode", extraParameter, cli.TerraformVariableFiles, cli.ActionParams);
+
+                if (resultCode == 1)
+                {
+                    commandResult.VerifySuccess();
+                }
+
+                Log.Info(
```

Comment:
Would these be better logged at `Verbose`?

Reply:
This is how the original script on the server was logging it, so I didn't want to introduce braking changes